eng/common/testproxy/scripts/tag-merge/README.md

```diff
@@ -11,6 +11,7 @@ This script merely allows the abstraction of some of this "combination" work.
 
 ### PreReqs
 
+- Must have []`pshell 6+`](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows)
```

Comment:
This can be enforced in the script itself as well to fail fast, see [example](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/TestResources/New-TestResources.ps1#L6-L7)